### PR TITLE
Use Chef 13

### PR DESCRIPTION
**Pull request by madwort**
* also move tests from Rackspace to AWS
* remove mysql functionality (easier than fixing it)

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -15,3 +15,5 @@
 default[:redmine][:db][:hostname] = 'localhost'
 default[:redmine][:db][:dbname]   = 'redmine'
 default[:redmine][:db][:username] = 'redmine'
+
+default['postgresql']['pg_gem']['version'] = '0.21.0'
```
Comment (madwort) [inline attributes/default.rb:19]: required setting for `postgresql-cookbook` version `6.1.1`

File: metadata.rb [hunk 2]
```diff
@@ -9,17 +9,17 @@
...
 depends 'database', '~> 4.0.9'
-depends 'postgresql'
-depends 'mysql', '< 6.0' # before recipes removed
-depends 'mysql-chef_gem', '< 1.0' # transitive dependency, before recipes removed
+depends 'postgresql', '~> 6.1.1'
 depends 'sqlite'
 depends 'certificate'
 depends 'iptables'
 depends 'logrotate', '~> 1.9.2'
+depends 'build-essential', '~> 2.2.4'
```
Comment (madwort) [inline metadata.rb:21]: pinned to avoid an incompatibility in `build-essential` latest (iirc `8.x.x`)

File: metadata.rb [hunk 3]
```diff
@@ -9,17 +9,17 @@
...
+depends 'chef-vault', '~> 2.1.0'
```
Comment (madwort) [inline metadata.rb:22]: pinned to avoid an incompatibility present in `3.0.0`

File: attributes/default.rb (hunk 1 above)
Comment (madwort) [reply attributes/default.rb:19]: https://github.com/sous-chefs/postgresql/issues/480

File: metadata.rb (hunk 2 above)
Comment (madwort) [reply metadata.rb:21]: (use same version as deployment)

File: .kitchen.ci.yml [hunk 4]
```diff
@@ -1,17 +1,26 @@
...
   name: chef_zero
   data_path: test/shared
 
+verifier:
+  name: busser
+  version: 0.6.0
+
+common: &common
+  roles_path: roles
+  encrypted_data_bag_secret_key_path: 'test/integration/postgresql/data_bag_key'
```
Comment (blaskov) [inline .kitchen.ci.yml:22]: Why don't you decrypt all test data bags and skip this?

**Review by blaskov — APPROVED** [review]
I would have had more comments if this had been our own cookbook, but considering the fact it's 3rd party one and tests pass ok, I think it's good to go 👍

File: .kitchen.ci.yml (hunk 4 above)
Comment (madwort) [reply .kitchen.ci.yml:22]: ...it would be more work... 😮

**Pull request merged**